### ISSUE TITLE
'destroy' removes mounted data-containers

### ISSUE THREAD

**Issue by hrbu** [meetfinch/decking#86]
When calling ```destroy <cluster>```, decking removes the data-containers that are defined as  a ```"mount-from"``` dependency.

```javascript
"base.coredatastore": {
	"image": "coredatastore:1.0-SNAPSHOT",
	"port": ["5984:5984"],
	"mount-from": ["base.coredatastore_volume"]
},
"base.coredatastore_volume": {
	"image": "coredatastore_volume:1.0-SNAPSHOT",
  	"data": true
}
```
... means, in case of  ```destroy``` my data-container ```base.coredatastore_volume``` will be destroyed too.

When calling ```destroy``` I'd prefer to
* do not touch data-containers by default and
* have a flag, that signals "remove data-containers too" explicitly.

What do you think?

**Comment by makeusabrew** [comment]
I agree. I actually noted this down somewhere (found it: #58) but then forgot to implement it that way. Data container destruction should be opt-in only. Will get it in to 0.4.1 :)

**Comment by makeusabrew** [comment]
Fixed as of 0.5.0; can pass in `--include-data` to `decking destroy` to remove data volumes. Other options are `--include-persistent` (to delete any containers with a `persistent: true` setting) and `--include-volumes` :)
